packages/@okta/courage-dist/okta.js

```diff
@@ -4406,14 +4406,15 @@ var SchemaUtils = {
    * @param {String} appName The app name that's mapped to/from Okta
    * @return {String} the source attribute username type value
    */
+  // eslint-disable-next-line complexity
   getSourceUsernameType: function getSourceUsernameType(mappingDirection, targetName, appName) {
     var sourceUsernameType = this.USERNAMETYPE.NONE;
     /* eslint complexity: [2, 7] */
 
     if (mappingDirection === 'oktaToApp') {
       if (targetName === 'userName') {
         sourceUsernameType = this.USERNAMETYPE.OKTA_TO_APP;
-      } else if (targetName === 'cn') {
+      } else if (targetName === 'cn' && appName === 'active_directory') {
```

Comment:
Wanted to understand why this change here and can't be done at view level.

Reply:
I believe this is added by @chuanbozhang-okta  [here](https://github.com/okta/okta-ui/commit/2ab08aa789918a3f0c79c2ec069a14dac5ecbc88) in okta-ui courage. Since I am upgrading the version now, it is pulled into SIW only now.

Reply:
@praveendurai-okta can you create a jira and add a `TODO` comment here linking to that jira..

Reply:
[OKTA-472892](https://oktainc.atlassian.net/browse/OKTA-472892) - Assigned to [@Alex Ma](https://okta.slack.com/team/U7KC9FCQH), please add labels if necessary to bring it down in your team's board.
[OKTA-472894](https://oktainc.atlassian.net/browse/OKTA-472894) - Assigned to me [#eng-enterpriseauth](https://okta.slack.com/archives/C8WSD3TM4)to help alex's changes pushed to dependent package.

Reply:
I don't think this functionality is used in the SIW, so we should be ok. Alex to confirm